assets/tests/components/mapPage.test.tsx

```diff
@@ -606,10 +598,103 @@ describe("<MapPage />", () => {
           </StateDispatchProvider>
         )
 
-        expect(
-          screen.getByRole("generic", { name: /vehicle properties card/i })
-        ).toBeVisible()
+        expect(vehiclePropertiesCard.get()).toBeVisible()
+      })
+    })
+  })
+
+  describe("<RoutePropertiesCard />", () => {
+    test("renders when a vehicle's route shape is selected", async () => {
+      jest.spyOn(global, "scrollTo").mockImplementationOnce(jest.fn())
+
+      const route = routeFactory.build()
+      const routePattern = routePatternFactory.build({ routeId: route.id })
+      const vehicle = vehicleFactory.build({
+        routeId: route.id,
+        routePatternId: routePattern.id,
+      })
+
+      mockUseVehicleForId([vehicle])
+      mockUseVehiclesForRouteMap({
+        [route.id]: [vehicle],
+      })
+      ;(usePatternsByIdForRoute as jest.Mock).mockReturnValue({
+        [routePattern.id]: routePattern,
+      })
+
+      const { container } = render(
+        <RealDispatchWrapper
+          initialState={stateFactory.build({
+            searchPageState: {
+              selectedEntity: {
+                type: SelectedEntityType.Vehicle,
+                vehicleId: vehicle.id,
+              },
+            },
+            userSettings: {
+              ladderVehicleLabel: VehicleLabelSetting.VehicleNumber,
+            },
+          })}
+        >
+          <RoutesProvider routes={[route]}>
+            <MapPage />
+          </RoutesProvider>
+        </RealDispatchWrapper>
+      )
+      expect(vehiclePropertiesCard.get()).toBeVisible()
```

Comment:
**question:** how necessary is this assertion for this test?

Reply:
I'll reword this test - I think it is useful for testing that the VPC and RPC are mutually exclusive, but that isn't totally captured by what is here.